### ISSUE TITLE
Allow auto-update of room & capacity information when editing locations via the "Add Class Series" form

### ISSUE THREAD

**Issue by leetucker** [leetucker/django-danceschool#24]
Because locations' capacity and room information is currently loaded via data attributes on the <option> tags, and not via Ajax, when you edit a Location's information inline, the room and default capacity information doesn't update until the page is refreshed, which is confusing.

**Comment by leetucker** [comment]
OK, I looked into this some more.  When one adds or changes an instance, Django's admin does indeed call back to the parent window of the id and name of the Location that has been added or changed.  There might be a way to just fire off an Ajax call when this callback happens (by creating an event handler for whenever the select options for the location are modified).

However, I think that it may make sense to just override the default callback process entirely.  Here's how that would work:

- In `danceschool.core.admin`, in the LocationAdmin class Override the ModelAdmin methods `response_add()` and `response_change()` to pass along not just the name and id of the added/modified location, but also the capacity information and room listing for that location.  Also, change the methods to return a modified version of templates/admin/response_popup.html
- The response template `templates/admin/response_popup.html` just returns a simple HTML document with included Javascript from static/admin/js/popup_response.js.  Change this document to call a modified version of this file.
- The current version of `static/admin/js/popup_response.js` just calls the functions `dismissAddRelatedObjectPopup()` and `dismissChangeRelatedObjectPopup()` from static/admin/js/admin/RelatedObjectLookups.js.  Instead, call a modified version of these functions that not only adds/modifies the ID and name of the added/changed Location instance, but that also loads in the passed defaultCapacity and room data if it is provided.

I don't have time to take this on immediately, but we should definitely go ahead and get this bug fixed before the rooms branch is merged into master.
